### PR TITLE
completed dom-event-delegation exercise - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-04-06 at 11 57 15 AM

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', function (event) {
+  console.log('event.target.tagName', event.target.tagName);
+  console.log('event.target', event.target);
+  if (event.target.tagName === 'BUTTON') {
+    const $closest = event.target.closest('.task-list-item');
+    console.log($closest);
+    $closest.remove();
+    return $closest;
```
Comment (shawnkost) [inline dom-event-delegation/main.js]: Remove this, we don't want to return anything here.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Bryan!

File: dom-event-delegation/main.js (hunk 1 above)
Comment (bbrown949) [reply dom-event-delegation/main.js]: removed.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**